app/experimenter/experiments/tests/test_serializers.py

```diff
@@ -118,6 +120,14 @@ def test_serializer_outputs_expected_schema(self):
         )
 
 
+class TestStatusSerializer(TestCase):
```

Comment:
This should probably be `TestExperimentChangeLogSerializer` to be consistent

Reply:
Thanks for pointing that out!!! :D